main/main.cpp

```diff
@@ -1664,6 +1665,8 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			OS::get_singleton()->disable_crash_handler();
 		} else if (I->get() == "--skip-breakpoints") {
 			skip_breakpoints = true;
+		} else if (I->get() == "--ignore-error-breaks") {
+			ignore_error_breaks = true;
```

Comment:
You need to add this to the `--help` output too.

Reply:
`--skip-breakpoints` is not in `--help` either (and probably some other debug-related options).

Reply:
I think we should add them then. We have a section of the `--help` that's specific to debugging, which this is a good fit. The description should likely clarify this is only effective if the debugger is connected.